crypto/rand/rand_vms.c

```diff
@@ -148,12 +465,19 @@ size_t rand_pool_acquire_entropy(RAND_POOL *pool)
     /*
      * Try not to overfeed the pool
      */
-    if (total_length > bytes_remaining)
+    if (total_length > bytes_remaining) {
```

Comment:
On a vaguely related **side note** one can actually wonder if this is right. I mean what's "overfeed the pool"? You don't want to overfeed the pool with entropy, right? So if you make an estimate on your source, wouldn't it be appropriate to adjust amount of data you pass down? Basically question is if "bytes_remaning" is an expression of how much entropy is missing, or is it actually how much data pool can accommodate? It's kind of ambiguous...

Reply:
`bytes_remaining` is the maximum available space in the pool, that's what `rand_pool_bytes_remaining()` returns.  That and the fact that `rand_pool_add()` will refuse the input entirely if it's larger than the available space in the pool should answer your question.

(on might argue that `rand_pool_add()` should accept the amount that it can, but that's outside of the scope of this PR)